Phaethon/Tests/IntegrationTests/LoginTest.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Data.Entity;
+using System.Data.Entity.Core.Objects;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Runtime.ExceptionServices;
+using System.Text;
+using System.Threading.Tasks;
+using Core.Model;
+using InternalApi.DataAccess;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System.Security.Cryptography;
+using InternalApi.DataManagement;
+
+namespace Tests.IntegrationTests
+{
+
+   public class LoginTest : InternalTestFakeServerBase
+    {
+        private Login userModel;
+        private Login userModel2;
+        private int count1;
+        private int count2;
+        private string name;
+        private string name2;
+        private string OGPass;
+        private LoginDa loginDa;
+
+        private void InitializeData()
+        {
+            userModel = new Login { Username = "Subject1", Password = "12355557" };
```

Comment:
Make your tests independent.

Right now, if one of your tests fails for some reason, Cleanup() will not be reached, and because Username is unique, all subsequent tests will fail.

Perhaps generating new random usernames, for each test would be a good idea.

Reply:
Or you could wrap your tests in try{} statements and put the Cleanup() in final{}

Or put attibute [TearDown] on your Cleanup() method, so itll be called everytime a test ends, no matter if it fails, encounters error or passes

Or find another way, you're free to do whatever you want.